docs/components/debug.md

```diff
@@ -0,0 +1,38 @@
+---
+title: debug
+type: components
+layout: docs
+parent_section: components
+---
+
+The debug component enables component-to-DOM serialization.
+
+## Example
+
+```html
+<a-scene debug></a-scene>
+```
+
+## Component-to-DOM Serialization
+
+By default, for performance reasons, A-Frame does not update the DOM with component data. If we open the browser's DOM inspector, we will see that many entities will have only the component name visible:
```

Comment:
@dmarcos explained to me when exactly the perf is impaired. `JSON.parse` + `JSON.stringify` being costly with animations is an example. that's probably a technical detail I'm just curious about, but now I'm not as surprised or confused.